wnfs/README.md

```diff
@@ -29,8 +29,6 @@
 
 <div align="center"><sub>:warning: Work in progress :warning:</sub></div>
 
-##
-
```

Comment:
This was added to visually separate the two sections. Can it be added back?

Reply:
ah, got it, i was reading in markdown and confused, yep